src/__tests__/standalone-syntax-test.js

```diff
@@ -133,3 +133,87 @@ test("standalone with syntax set by extension", t => {
 
   t.end()
 })
+
+test("standalone with path to custom parser", t => {
+  const config = {
+    rules: {
+      "block-no-empty": true,
+    },
+  }
+
+  standalone({
+    config,
+    syntax: `${fixturesPath}/custom-parser`,
+    code: ".foo { width: 200px; }\n.bar {}",
+    formatter: stringFormatter,
+  }).then(({ output }) => {
+    const strippedOutput = chalk.stripColor(output)
```

Comment:
Instead of reading the `output`, stripping the colors, and using `indexOf`, I suggest using `.then(({ results }) => {`: `results` will be the JSON stylelint results.